eduNEXT/xblock-filesmanager
# feat: allow to upload file with same name

### Description

This PR allows to uploading of files with the same name:

![image](https://github.com/eduNEXT/xblock-filesmanager/assets/33465240/ea7d7078-9565-42c2-a48a-c8c5f9c32a35)

### Testing instructions

1. Upload a file to the xblock
2. Save and re-upload the same file
3. The file should be updated without errors and the paths and the new file name should include an count of the files.
**Merge checklist:**
Check off if complete *or* not applicable:
